

fs/cachefs/status.c:54: error: conflicting types for `cachefs_status_open'
fs/cachefs/status.c:23: error: previous declaration of `cachefs_status_open'


Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/fs/cachefs/status.c |    2 +-
 1 files changed, 1 insertion(+), 1 deletion(-)

diff -puN fs/cachefs/status.c~cachefs-build-fix fs/cachefs/status.c
--- 25/fs/cachefs/status.c~cachefs-build-fix	2004-09-06 23:36:21.000000000 -0700
+++ 25-akpm/fs/cachefs/status.c	2004-09-07 02:30:14.691349488 -0700
@@ -50,7 +50,7 @@ struct file_operations cachefs_status_fi
 /*
  * open a status file
  */
-static ssize_t cachefs_status_open(struct inode *inode, struct file *file)
+static int cachefs_status_open(struct inode *inode, struct file *file)
 {
 	struct seq_file *m;
 	int ret;
_
